docs/conventions/architecture.md

```
@@ -0,0 +1,57 @@
+# 폴더 구조 & 기술 스택
+
+## 폴더 구조
+
+`shared/` 아래 폴더는 전부 common(공통)의 의미로 사용한다.
+`pages/` 아래 세부 폴더(components, hooks 등)는 각 페이지에 종속된다.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)